### PR TITLE
document scoop/brew installations & completions

### PR DESCRIPTION
**Pull request by svix-frank**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,9 +14,26 @@ A CLI to interact with the Svix API.
...
 ## Installation
 
-Homebrew Tap & Scoop Bucket Coming soon.
+### macOS
 
-For now, you can download the binary executable and use it straight away without having to install any additional dependencies.
+The Svix CLI is available on macOS via [Homebrew](https://brew.sh/):
+
+```sh
+brew install svixhq/svix/svix
```
Comment (tasn) [inline README.md:22]: Is it missing the `tap add` or is it good like this?

**Review by tasn — APPROVED** [review]
Looks good to me. Just made one small comment.

File: README.md (hunk 1 above)
Comment (svix-frank) [reply README.md:22]: It's good like this, this is just a oneliner,`brew install <tap>/<fomula>`
the alternative would be
```sh
brew tap svixhq/svix
brew install svix
```

Comment (tasn) [reply README.md:22]: Cool. :)

**Pull request merged**